packages/x-date-pickers/src/internals/hooks/useStaticPicker/useStaticPicker.tsx

```diff
@@ -48,7 +48,10 @@ export const useStaticPicker = <
         <PickerStaticViewLayout
           {...layoutProps}
           components={components}
-          componentsProps={componentsProps}
+          componentsProps={{
+            ...componentsProps,
+            actionBar: undefined,
+          }}
```

Comment:
I don't get why you need to override the `actionBar` props with `undefined`.

Reply:
Otherwise TS complains that the types of the props of the action bar used by `PickerStaticViewLayout` and `PickersViewLayout` are note the same.

And since the action bar is not handled by `PickerStaticViewLayout` on the new picker, I'm passing `undefined` to bypass the problem.
I could also `@ts-ignore` but I did not want to risk forgetting to remove it once we drop the `actionBar` from `PickerStaticViewLayout` altogether.

Same for the mobile / desktop wrappers.

Reply:
Oh I went too fast... this `actionBar: undefined` only applies on mobile and desktop pickers, on static we don't have any "legacy" wrapper..

Thanks for the catch !